opencga-app/src/main/java/org/opencb/opencga/app/cli/main/executors/AnalysisClinicalCommandExecutor.java

```diff
@@ -121,19 +127,19 @@ public void execute() throws Exception {
             case "interpretation-info":
                 queryResponse = infoInterpretation();
                 break;
-            case "interpreter-cancer-tiering-run":
+            case CANCER_TIERING_INTERPRETATION_RUN_COMMAND:
```

Comment:
This is an autogenerated code, so this change will be removed in the future. Doesn't matter, as the change will be equivalent.

Reply:
That's very true, @jtarraga let's discuss this